docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/hpc-launching-architecture.rst

```diff
@@ -0,0 +1,370 @@
+
+#. Upon being scheduled by the workload manager, the batch script is executed and tasks are launched on the allocated compute nodes.
+      #. One Determined environment image is started per node (``srun``, ``pbs_tmrsh``) using the
+         specified container runtime (Apptainer, Singularity, Podman, Enroot).
+
+      #. During container initialization, per-rank local directories are configured using soft links
+         (``/tmp``, ``/run/determined``), and the container stdout/stderr starts streaming output
+         directly to the Determined master.
+
+      #. For distributed training, horovodrun is invoked within the container to distribute work
+         within multiple processes across the containers.
+
+#. Once started, the job ID from the workload manager is displayed in the experiment log.
+
+#. Determined polls the launcher for job status waiting for completion.
+
+#. Determined requests the launcher to clean up the job. File ownership is transferred back to the
+   launcher (``sudo chmod``) which then deletes all temporary files and logs associated with the
+   job.
+
+*************************
+ Job Resource Scheduling
+*************************
+
+As noted earlier, when using the HPC launcher, Determined delegates all job scheduling and
+prioritization to the HPC workload manager (Slurm/PBS). Resource requests are generated into the
+submitted batch file derived from the following Determined configuration attributes.
+
+Cluster Configuration
+=====================
+
+The :ref:`resource_manager <cluster-configuration-slurm>` section of the cluster configuration
+contributes the following resource scheduling configuration. See the :ref:`slurm/pbs section
+<cluster-configuration-slurm>` of the cluster configuration reference for the full list of
+configuration options.
+
+-  ``slot_type``: The default slot type (``cuda``, ``rocm``, ``cpu``) assumed when users request
+   resources from Determined in terms of ``slots_per_trial``. May be overridden per
+   partition/resource pool.
+
+-  ``tres_supported``: Indicates if ``SelectType=select/cons_tres`` is set in the Slurm
+   configuration. It affects how Determined requests GPUs from Slurm. The default is true, which
+   allows an experiment to request GPU ``slots_per_trial`` without having to specify
+   ``slurm.slots_per_node``, and Slurm will automatically allocate the number of nodes needed to
+   satisfy the number of requested GPU slots.
+
+-  ``gres_supported``:
+      -  For Slurm, it indicates that ``GresTypes=gpu`` is set in the Slurm configuration, and nodes
+         with GPUs have properly configured GRES indicating the presence of any GPUs. The default is
+         true. When false, Determined will request ``slots_per_trial`` nodes and utilize only GPU 0
+         on each node unless slurm.slots_per_node is specified. It is the user’s responsibility to
+         ensure that GPUs will be available on nodes selected for the job using other configurations
```

Comment:
I "think" that a comma is needed after "configurations".

Reply:
Dropped the details.